controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/screwdriver/ScrewdriverApiHandler.java

```diff
@@ -90,8 +92,7 @@ private HttpResponse trigger(HttpRequest request, String tenantName, String appl
             // Since this is a manual operation we likely want it to trigger as soon as possible so we add it at to the
             // front of the queue
             application = controller.applications().deploymentTrigger().triggerAllowParallel(
-                    jobType, application, true, true,
-                    "Triggered from screwdriver/v1"
+                    new DeploymentTrigger.Triggering(application, jobType, true, "Triggered from screwdriver/v1"), Collections.emptySet(), true
```

Comment:
Ah, this turned out wrong, I see. `concurrentlyWith` should be all jobs here, or the force thing should make the test be skipped. I'll do the latter, I think.